_extras/design.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: page
+title: "Lesson Design"
+permalink: /design/
+---
+## Stage 0 - Assumptions
+
+*   Audience
+    *   Graduate students and practicing scientists in numerate disciplines from cosmology to economics
+        *   Lesson content broadly applicable across disciplines
+        *   Lesson examples from population demography ([Gapminder data][gapminder-data]) 
+    *   Must understand basic data management and manipulation 
+        *   Likely worked with spreadsheets and interactive stats tools (SAS)
+        *   Familiar with tabular data structure
+        *   Familiar with filtering and summarizing data in groups
+        *   Understands basic statistics (mean, stdev, lm) to compare groups
+    *   Should have some limited programming experience in R, including:
+        *   How to load a CSV (`read.csv()`) 
+        *   Basic `dplyr` function (`filter()`, `summarize()`, etc.)
+        *   Best preparedness benchmark from [r-novice-gapminder lesson][r-novice-gapminder]
```

Comment:
I'm not sure myself. Using `read.csv()` and `dplyr` are just a few episodes in the `r-novice-gapminder` lesson, which is why I'm having trouble here with the language. I think it will be good to point students to the resource as a prereq. But, it contains more information than we would require as I've currently outlined as 'limited programming experience'. Maybe a solution is to link directly to the episodes being referenced.

Alternately, we could set the benchmark at completion of `r-novice-gapminder`. It would be easy to include control flow, functions, reports, etc. to make this lesson more advanced in content. I don't have strong feelings about what level the coding expectations are and think I can craft a solid lesson at both levels.